### ISSUE TITLE
angular2-universal-preview

### ISSUE THREAD

**Issue by A77AY** [AngularShowcase/github-api-sample-app#1]
Why you do not use [angular2-universal-preview](https://www.npmjs.com/package/angular2-universal-preview)?

**Comment by mgechev** [comment]
Yeah, most likely I will include universal once the functionality of the application is completed :-)
